src/ui/Modal/Modal.tsx

```diff
@@ -48,7 +51,7 @@ const Modal: FC<ModalProps> = ({
             {closeButtonIcon ?? <Icon icon={closeFill} tw="w-6 h-6" />}
             <span tw="sr-only">Close dialog</span>
           </ButtonGhost>
-          <Text component="h3" variant="h5" id={`${id}-modal-title`}>
+          <Text component="h3" variant="h5" id={`Modal-Title-${modalId}`}>
```

Comment:
That's a weird naming convention. Pascal Kebab?

Reply:
Idk, never thought of it 😂

Switched to Pascal.

Reply:
This is worse... do it the same as classnames: lower-kebab-case.

Reply:
PascalCase is fine too.

Reply:
But not both.

![image](https://user-images.githubusercontent.com/52666982/145519463-fd7ccaba-f219-4c88-b663-10eb2e31f754.png)

Reply:
> This is worse... do it the same as classnames: lower-kebab-case.

Updated, generally I would have different convention between id and className but considering we don't use className much here, I can use the `lower-kebab-case`